komorebi-bar/src/config.rs

```diff
@@ -178,3 +185,26 @@ pub enum LabelPrefix {
     /// Show an icon and text
     IconAndText,
 }
+
+#[derive(Copy, Clone, Debug, Serialize, Deserialize, JsonSchema)]
+pub struct AlphaColour {
+    /// Color
+    pub color: Option<Colour>,
+    /// Alpha value for the color transparency [[0-255]] (default: 200)
+    pub transparency_alpha: Option<u8>,
+}
+
+impl From<AlphaColour> for Color32 {
+    fn from(value: AlphaColour) -> Self {
+        let color = match value.color {
+            Some(color) => color.into(),
+            None => Color32::from_rgb(0, 0, 0),
```

Comment:
This should probably default to the theme's background color

Reply:
Ah good call, then it could even apply alpha to that! 👍

Reply:
This should now be possible 👍